source/clear-linux/guides/maintenance/enable-user-space.rst

```diff
@@ -4,38 +4,35 @@ Create and enable a new user space
 ##################################
 
 This section provides steps to complete the following basic setup tasks for
-a newly installed |CL| system:
+a newly installed |CLOSIA| system:
 
-* Create a new user
-* Update the OS to its most current version using `swupd`.
-* Install the most common applications for system administrators and
-  developers using bundles.
-* Setup a new user.
-* Add the new user to the `wheel` group.
-* Install a GUI using those `sudo` privileges.
+* Create a new user.
+* Install and update the OS software to its current version.
```

Comment:
Let's revert this line to the original. The swupd command is relevant here.
* Update the OS to its most current version using `swupd`.